components/StyledCard.tsx

```diff
@@ -31,8 +30,7 @@ type StyledCardProps = BackgroundProps &
   SpaceProps &
   TypographyProps &
   ShadowProps &
-  ColorProps &
-  React.HTMLProps<HTMLDivElement>;
```

Comment:
Why remove this?

Reply:
With this annotation, the width prop on StyledCard was not accepting a responsive value `[number, number, number]`

Reply:
Could have been interesting to do:

```
Omit<React.HTMLProps<HTMLDivElement>, 'width>
```

It's still valid to pass down HTML attributes to this comp

Reply:
I think html attributes are still valid, I believe they are already added by the `styled.div` typing. 
Without this added notation, I see React.HTMLAttributes in the types for this component, for example:

`<StyledCard translate="nope" />`

`translate?: 'yes' | 'no' | undefined;
                 ~~~~~~~~~
    The expected type comes from property 'translate' which is declared here on type 'IntrinsicAttributes & { borderWidth?: ResponsiveValue<string | number | symbol, Required<Theme<TLengthStyledSystem>>>; ... 360 more ...; borderY?: ResponsiveValue<...>; } & { ...; } & { ...; }'`